test/Test/Smoke.hs

```diff
@@ -448,13 +426,13 @@ nettestImplTzbtcClient (NettestEnv env _) = NettestImpl
             TZBTCTypes.TransferOwnership (arg #newOwner -> newOwnerAddress) ->
               callTzbtc [ "transferOwnership" , toString $ formatAddress newOwnerAddress ]
             TZBTCTypes.AcceptOwnership _ -> callTzbtc $ [ "acceptOwnership" ]
-            _ -> niTransfer impl td
-          _ -> niTransfer impl td
-        Nothing -> niTransfer impl td
+            _ -> niTransfer impl s td
+          _ -> niTransfer impl s td
+        Nothing -> niTransfer impl s td
```

Comment:
I was going to ask what was the point of adding `$` before realizing it was an `s`.

So, minor nitpick, but I suggest changing the argument match to just `sender` (or something similar) and using it here (and use `unSender sender` instead of `senderAddr` below).

Reply:
I think you missed this one :)

Reply:
Oh, indeed